### PR TITLE
Slightly remaps and adds more supplies to the animal hospital ruin

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
tweak: The lavaland animal hospital ruin has been remapped, including more supplies as well as light sources.
/:cl:

DM screenshot:

List o' changes:
* Removed the animal spawners. Nobody ever used them, they were completely useless except to die, and they couldn't even *breathe* in lavaland's atmosphere and so had a life expectancy akin to that of an ant being drowned.
* Replaced all of the doors with normal variants, that close on their own.
* Moved the rejuvenation pods (still only 3, the other 2 are normal sleepers) to a new room that has some first aid supplies and a Medibot that heals when the target has 40 or more damage.
* Added two more health HUDs to the locker, so each doctor can have one.
* Added three blue glowsticks and a stack of 30 titanium to the supply closet, for light and fixing walls respectively.

This should make the vet ruin a bit less neglected and unpolished, and a bit more fun and feasible to actually play as.

### CODE REVIEW

**Comment by GunHog** [comment]
What exactly is fun about animal hospital? Maybe we should turn it into some sort of surgery tutorial ruin?

**Comment by InsaneHyena** [comment]
Augmentation clinic.

**Comment by Armhulen** [comment]
Maybe we should remove it

**Comment by Repukan** [comment]
This ruin should be removed, no amount of tweaking will save it from its own uselessness. 

Also getting rid of the cancer rat goes against the core of the ruin.

**Comment by Xhuis** [comment]
The only core thing that's cancer about the ruin is the ruin itself, not the rat.

At any rate, I view it as a deliberately challenging lavaland role with very few supplies but that's very capable. With enough patience you can get bone tools and weapons, and so be as good of a hunter as an ash walker. I don't feel like it should be removed, though I suppose it's always an option.

**Comment by Repukan** [comment]
When you have a sick horse, you don't put crutches and bandages on his leg and extend his pain.

You take them behind the shed.

**Comment by Xhuis** [comment]
You're free to remove it if you'd like. I play the role often enough that I find it very much doable, just challenging. The only real gripe I had with the first iteration was the lack of light sources and the fact that doors didn't close on their own; I just did some minor touching up while I was in the neighborhood.

**Comment by Bawhoppen** [comment]
It's as good as any other ruin (not that other ruins are especially good). No reason to remove it.

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -249,6 +249,11 @@
 	everyone? Where did they go? What happened to the hospital? And is that <i>smoke</i> you smell? You need to find someone else. Maybe they can tell you what happened.</b>"
 	assignedrole = "Translocated Vet"
 
+/obj/effect/mob_spawn/human/doctor/alive/lavaland/Destroy()
+	var/obj/structure/fluff/empty_sleeper/S = new(get_turf(src))
```
Comment (Cyberboss) [inline code/game/objects/structures/ghost_role_spawners.dm]: get_turf(src) -> drop_location()

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Above

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**